src/Config.php

```diff
@@ -1402,6 +1402,12 @@ function formatGlobalLayout(layout) {
             echo "</tr></table>";
 
             echo "</td></tr>";
+
+            echo "<tr class='tab_bg_2'>";
+            echo "<td><label for='dropdown_use_flat_dropdowntree_on_search_result$rand'>" . __('Display the tree dropdown short name on search result') . "</label></td><td>";
+            Dropdown::showYesNo('use_flat_dropdowntree_on_search_result', $data["use_flat_dropdowntree_on_search_result"], -1, ['rand' => $rand]);
```

Comment:
The existing `use_flat_dropdowntree` preference is meant to `Display the complete name in tree dropdowns`.
It seems that this one will act the opposite. It may lead to confusion/mistakes.

Reply:
reverted